google-cloud-bigtable/test/google/cloud/bigtable/table/column_families_test.rb

```diff
@@ -37,7 +37,7 @@
   it "modifies column families in the table" do
     modifications = [
       Google::Bigtable::Admin::V2::ModifyColumnFamiliesRequest::Modification.new(
-        id: "cf1",
+        id: "cf4",
```

Comment:
I had to make these changes (and the one like this in the documentation) because ColumnFamilyMap raises when adding a name that already exists, and when updating or deleting a name that does not exist.